source/docs/3/basic_filtering_and_styling_with_expressions.rst

```diff
@@ -0,0 +1,120 @@
+Basic Filtering and Styling with Expressions (QGIS3)
+====================================================
+
+Expressions offer a powerful way to manipulate attribute values, geometries and variables to dynamically modify map features on the fly. This tutorial is an introduction to some common expressions and how they are used in map making. 
+
+.. note::  
+Tutorial contributed by: `Steven Kim <https://geographyclub.github.io/>`_
```

Comment:
block after note needs to be indented to take effect. Anyway for this one, move the contribution text right below the title with the following rst text
```
*Contributed by:* `Steven Kim <https://geographyclub.github.io/>`_
```